File: radialbuttonmenu/src/main/java/huberd/laurent/radialbuttonmenu/Translation/LayoutManager.java
package huberd.laurent.radialbuttonmenu.Translation;

import android.support.design.widget.FloatingActionButton;

import huberd.laurent.radialbuttonmenu.RadialMenu;

public interface LayoutManager {
    Translation getMainButtonTranslation(RadialMenu radialMenu, FloatingActionButton floatingActionButton);
    Translation getSecondaryButtonTranslation(RadialMenu radialMenu, int index, FloatingActionButton floatingActionButton);
}

File: radialbuttonmenu/src/main/java/huberd/laurent/radialbuttonmenu/Translation/LayoutManager.kt
package huberd.laurent.radialbuttonmenu.Translation

import android.support.design.widget.FloatingActionButton

import huberd.laurent.radialbuttonmenu.RadialMenu

interface LayoutManager {
    fun getMainButtonTranslation(radialMenu: RadialMenu, floatingActionButton: FloatingActionButton): Translation
    fun getSecondaryButtonTranslation(radialMenu: RadialMenu, index: Int, floatingActionButton: FloatingActionButton): Translation
}
